src/services/store-service.js

```diff
@@ -46,41 +40,39 @@
                     return;
                 }
 
-                var oldArray = order(favouriteStocks);
-                var fromIndex = oldArray.indexOf(fromItem);
-                var toIndex = oldArray.indexOf(toItem);
-                oldArray.splice(toIndex, 0, oldArray.splice(fromIndex, 1)[0]);
+                var window = getWindowStore();
```

Comment:
Fairly nitpicky, but I think it'd be better to rename `window` to something else (`windowStore`?) to avoid ambiguity with the native object.